Makefile

```diff
@@ -51,6 +51,10 @@ clean:
 .PHONY: html
 html:
 	$(SPHINXBUILD) -b html $(ALLSPHINXOPTS) $(BUILDDIR)/html
+
+	# support for Cloudflare Pages' redirects
+	@cp _redirects build/html/_redirects
```

Comment:
Instead of this, you can add the file directly to `source/_static` and Sphinx automatically adds it to the build output

Reply:
Oh, sweet, lemme try that.

Reply:
Sadly, does not seem to work. At least on my local the file gets put into `_static/` output folder, not into root. 

<img width="211" alt="Screenshot 2022-10-22 at 15 49 20" src="https://user-images.githubusercontent.com/311580/197342664-62bd708c-30bd-4882-a0eb-d06b8e550af5.png">

Reply:
Never mind